.github/workflows/ci.yaml

```diff
@@ -231,8 +240,20 @@ jobs:
     runs-on: ${{ github.actor == 'dependabot[bot]' && 'ubuntu-latest' || 'ubuntu-latest-large' }}
 
     steps:
-      - name: Free disk space
+      - name: Free Disk Space (Ubuntu)
         uses: jlumbroso/free-disk-space@v1.3.1
+        with:
+          # Runs quick, run rm in background
+          tool-cache: true
+
+          # These run slower as use apt to uninstall packages
+          # Turned on by default, so we disable them
+          android: false
+          dotnet: false
+          haskell: false
+          large-packages: false
+          docker-images: false
+          swap-storage: false
```

Comment:
```suggestion
      - name: Free disk space
        uses: ./.github/actions/free-disk-space
```

Reply:
wouldn't work as we don't checkout repo in this step